variables.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
why is this in a separate file?

Reply:
because there are several in-memory variables I would used to store data - like producers, consumers, quene, etc. And putting them in the kafkaserver.py file will make it look kind of rowdy. That's why I thought to separate them in a new file. I don't know if that's okay?